src/components/Calculator.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import calculate from './logic/calculator';
+
+function Calculator() {
```

Comment:
Good job jobs with Hooks, but we need to make use of ES6 i.e Arrow function.

Example:

```
const Calculator = () => {
......
}
```